atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionTest.java

```diff
@@ -1184,6 +1226,7 @@ public void testRowsColumnRangesSingleIteratorVersion() {
         runTestForGetRowsColumnRangeSingleIteratorVersion(10, 10, 5);
         runTestForGetRowsColumnRangeSingleIteratorVersion(10, 10, 10);
         runTestForGetRowsColumnRangeSingleIteratorVersion(100, 100, 99);
+        runTestForGetRowsColumnRangeSingleIteratorVersion(100, 11, 0);
```

Comment:
can you comment why this test is there?

Reply:
done! Wanted to test when a row's cells are spread across multiple batches